connaisseur/flask_application.py

```diff
@@ -153,6 +153,17 @@ async def __validate_image(type_index, image, admission_request):
     type_, index = type_index
     logging_context.update(image=original_image)
 
+    # a resource that is being updated, without any image reference changes,
+    # can be automatically skipped, as image validation happened at some
+    # point prior
+    update_approval_on = os.environ.get("AUTOMATIC_UPDATE_APPROVAL", "0") == "1"
+    if update_approval_on and admission_request.operation.upper() == "UPDATE":
+        old_images = admission_request.old_wl_object.containers.values()
+        if image in old_images:
+            msg = f'automatic update approval for "{original_image}".'
```

Comment:
If I'd read that log msg the auto update would trigger different (and wrong) associations in my mind. None of them make particular sense with Conny, but maybe we can find a better name for either the feature or at least the log msg

For log should be easy to be bit more verbose, for feature maybe sth like auto_approve_unchanged

Reply:
did go for `automatic unchanged approval` to stay consistent with `automatic child approval` + changed log message.